src/main/java/com/pinterest/secor/tools/LogFileDeleter.java

```diff
@@ -44,7 +44,8 @@ public void deleteOldLogs() throws Exception {
         }
         String[] consumerDirs = FileUtil.list(mConfig.getLocalPath());
         SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss z");
-        format.setTimeZone(TimeZone.getTimeZone("UTC"));
+        TimeZone timeZone = mConfig.getTimeZone() != null ? TimeZone.getTimeZone(mConfig.getTimeZone()) : TimeZone.getTimeZone("UTC");
```

Comment:
Looks like this piece of code is also repeated here, probably better to move that into a helper method.